app/schemas/grade_report.py

```diff
@@ -0,0 +1,32 @@
+from typing import Optional
+from pydantic import BaseModel
+from datetime import datetime
+
+class GradeReportSchema(BaseModel):
+    id: int
+    average: float
+    median: float
+    minimum: float
+    maximum: float
+    stdev: float
+    scores: list[float]
+    total_points: float
+    num_submitted: int
+    # num_skipped = num_submitted (== num students w/ submissions) - num_active_submissions_already_graded
```

Comment:
Any reason for this to be here? If so I think it should have a comment explaining why it's commented out

Reply:
it's not commented out, it is a comment

Reply:
Oh gotcha, valid